next.config.js

```diff
@@ -0,0 +1,44 @@
+let config = {
+  typescript: {
+    // !! WARN !!
+    // Dangerously allow production builds to successfully complete even if
+    // your project has type errors.
+    // !! WARN !!
+    ignoreBuildErrors: true
+  },
+  async rewrites() {
+    return [
+      {
+        source: "/api/configs_db/:path*",
+        destination:
+          "https://incident-commander.canary.lab.flanksource.com/config/db/:path*"
+      },
+      {
+        source: "/api/canary/:path*",
+        destination:
+          "https://incident-commander.canary.lab.flanksource.com/canary/:path*"
+      },
+      {
+        source: "/api/incidents_db/:path*",
+        destination:
+          "https://incident-commander.canary.lab.flanksource.com/db/:path*"
+      },
+      {
+        source: "/api/apm/search/:path*",
+        destination:
+          "https://incident-commander.canary.lab.flanksource.com/apm/search/:path*"
+      },
+      {
+        source: "/api/api/:path*",
```

Comment:
Why /api/api ?  whats the different between this and /api/canary ?

Reply:
it's just mapping to existing paths. E.g. canary backend will receive /api/canary. The `api` prefix is for front-end and ui-backend to differentiate api calls. E.g. `/api/api` means canary request. etc.